index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Inter:wght@300&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+    <title>Tsige's Personal Portifolio</title>
+</head>
+  <body>
+    <nav class="toolbar">
+        <div class="img">
+            <img src="images/mobile.png" alt="header-image">
+        </div>
+      <!-- <div class="logo"><a href="">Tsige Wossne</a></div> -->
+      <div class="menu_main">
+        <input type="checkbox" class="toggler">
+        <div class="humbuger"></div>
+        <div class="menu">
+          <div class="overlay">
+            <div>
+              <ul>
+                <li><a href="">Home</a></li>
+                <li><a href="">Projects</a></li>
+                <li><a href="">Contribution</a></li>
+              </ul>
+            </div>
+          </div>
+        </div>
+      </div>
+    </nav>
+    <section class="headline">
+      <div class="container">
+        <h1>Hey there. <br>
+            I’m Tsige.</h1>
+        <h1 class="green">I’m a software developer! </h1>
+           <p>I can help you build any software as your need
+            including DesktopApp ,Mobile feature or website.
+            For more Please Look through 
+            some of my work and experience that I have!</p>
+        <h3>
+            <a href="">Let's Contact</a> 
+        </h3>  
```

Comment:
- [ ] Kindly, make sure you remove this line, because, it is not a part of the design 😃